app/root/root.css

```diff
@@ -601,6 +601,14 @@ svg.icon.orange {
   user-select: none;
 }
 
+.card .all {
```

Comment:
nit: You can reuse the existing className (`more`) to avoid introducing new styles.

Reply:
Done!